### ISSUE TITLE
Error when opening "wallet fusion settings"

### ISSUE THREAD

**Issue by acidsploit** [Electron-Cash/Electron-Cash#2393]
Running EC from source @ git HEAD

I get the following error when opening "wallet fusion settings".

![Screenshot_20220122_103436](https://user-images.githubusercontent.com/10303422/150673357-ac7cc0dc-8135-42fb-afd7-a476aad84186.png)

```|125.509| |00| [cashfusion2.flipstarter] refresh_all                                            
Traceback (most recent call last):
  File "/home/thomas/appimage/ec-githead/Electron-Cash/electroncash_plugins/fusion/qt.py", line 857, in show_wallet_settings
    win = WalletSettingsDialog(Plugin.get_suitable_dialog_window_parent(self.wallet),
  File "/home/thomas/appimage/ec-githead/Electron-Cash/electroncash_plugins/fusion/qt.py", line 1319, in __init__
    self.refresh()
  File "/home/thomas/appimage/ec-githead/Electron-Cash/electroncash_plugins/fusion/qt.py", line 1435, in refresh
    self.sb_selector_count.setValue(sel_count)
TypeError: setValue(self, int): argument 1 has unexpected type 'float'
```

**Comment by emergent-reasons** [comment]
Confirmed I get the same behavior on py 3.10.0 running from head 33975a8a5

**Comment by acidsploit** [comment]
Changing 

`self.sb_selector_count.setValue(sel_count)`

to

`self.sb_selector_count.setValue(round(sel_count))`

Seems to fix it. I don't know if this break it in other ways though. (Assuming round() always returns an int)

**Comment by cculianu** [comment]
Damn this PyQt brokenness on Py3.10. Anyway I merged #2394, should be fixed now.
